content/using-packages.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Using Packages
+---
+
+After reading this article, you'll know:
+
+1. The third party packages systems you can use with Meteor, NPM and Atmosphere
```

Comment:
This makes it sound like Meteor, NPM, and Atmosphere are package systems, maybe

"About NPM and Atmosphere, two JavaScript package systems you can use with Meteor"